apps/web/playwright/signup.e2e.ts

```diff
@@ -27,6 +27,12 @@ test.describe("Signup Main Page Test", async () => {
     await page.locator('[data-testid="continue-with-google-button"]').click();
     await page.waitForURL("/auth/sso/google");
   });
+
+  test("Continue with SAML button must exist / work", async ({ page }) => {
+    await page.goto("/signup");
+    await expect(page.locator('[data-testid="continue-with-saml-button"]')).toBeVisible();
+    await page.locator('[data-testid="continue-with-saml-button"]').click();
+  });
```

Comment:
We should skip this based on if SAML is enabled or not on the instance - this PR fails tests locally for me as saml is not enabled for me locally.